rskj-core/src/test/java/co/rsk/peg/BridgeUtilsTest.java

```diff
@@ -2914,4 +2919,206 @@ private Federation getGenesisFederationForTest(BridgeConstants bridgeConstants,
 
         return federation;
     }
+
+    @Test
+    public void getMinimumPegInTxValue() {
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        // Before RSKIP219 activation
+        when(activations.isActive(ConsensusRule.RSKIP176)).thenReturn(true);
```

Comment:
rskip176 is not involved in this operation

Reply:
Same here